google/cloud/privateca/README.md

```diff
@@ -38,22 +41,24 @@ this library.
 
 <!-- inject-quickstart-start -->
 ```cc
-#include "google/cloud/privateca/ EDIT HERE .h"
-#include "google/cloud/project.h"
+#include "google/cloud/privateca/certificate_authority_client.h"
 #include <iostream>
 #include <stdexcept>
 
 int main(int argc, char* argv[]) try {
   if (argc != 2) {
-    std::cerr << "Usage: " << argv[0] << " project-id\n";
+    std::cerr << "Usage: " << argv[0] << " project-id location-id caPool-id\n";
     return 1;
   }
 
   namespace privateca = ::google::cloud::privateca;
-  auto client = privateca::Client(privateca::MakeConnection(/* EDIT HERE */));
+  auto client = privateca::CertificateAuthorityServiceClient(
+      privateca::MakeCertificateAuthorityServiceConnection());
 
-  auto const project = google::cloud::Project(argv[1]);
-  for (auto r : client.List /*EDIT HERE*/ (project.FullName())) {
+  auto const ca_pool = "projects/" + std::string(argv[1]) + "/locations/" +
+                       std::string(argv[2]) + "/caPools/" +
+                       std::string(argv[3]);
+  for (auto r : client.ListCertificates(ca_pool)) {
```

Comment:
Seems like `ListCaPools` would have been easier to implement and to understand:

https://cloud.google.com/certificate-authority-service/docs/reference/rpc/google.cloud.security.privateca.v1#google.cloud.security.privateca.v1.ListCaPoolsRequest